angular.json

```diff
@@ -148,26 +105,29 @@
       "projectType": "application",
       "architect": {
         "build": {
-          "builder": "@angular-devkit/build-angular:browser",
+          "builder": "@angular-devkit/build-angular:application",
           "options": {
-            "outputPath": "dist/browser/iframe",
-            "index": "./site/iframe/index.html",
-            "main": "./site/iframe/main.ts",
-            "tsConfig": "./site/iframe/tsconfig.app.json",
-            "polyfills": "./site/iframe/polyfills.ts",
+            "outputPath": {
+              "base": "dist/browser/iframe",
+              "browser": ""
+            },
```

Comment:
Make sure to output ng-zorro-antd-iframe to the dist/browser/iframe directory. Since we're using the application builder, the output path will automatically get the browser subdirectory (outputPath/browser), but we don't need it, so we'll set it to empty.